addons/auth_oauth/static/src/scss/auth_oauth.scss

```diff
@@ -0,0 +1,19 @@
+.o_auth_oauth_providers {
+    font-size: 14px;
+
+    i[class*='facebook'] {
+        color: #3b5998;
+    }
+
+    i[class*='google-plus'] {
+        color: #de564a;
+    }
+
+    i.o_custom_icon {
+        margin: 0 0.15em;
+        @include size(1em);
+        border: 3px solid #875A7B;
+        border-radius: 100%;
+        transform: translateY(2px);
+    }
+}
```

Comment:
Just a detail but you'll need to configure your editor to add a new line at the end of files :wink:

Reply:
uops ^^